samples/api/hello_triangle/hello_triangle.cpp

```diff
@@ -207,11 +207,33 @@ void HelloTriangle::init_instance(Context &                        context,
 	std::vector<const char *> active_instance_layers(required_instance_layers);
 
 #ifdef VKB_VALIDATION_LAYERS
-	active_instance_layers.push_back("VK_LAYER_GOOGLE_threading");
-	active_instance_layers.push_back("VK_LAYER_LUNARG_parameter_validation");
-	active_instance_layers.push_back("VK_LAYER_LUNARG_object_tracker");
-	active_instance_layers.push_back("VK_LAYER_LUNARG_core_validation");
-	active_instance_layers.push_back("VK_LAYER_GOOGLE_unique_objects");
+	// Optimal validation layers
+	std::vector<const char *> default_debug_layers{{
+	    "VK_LAYER_KHRONOS_validation",
+	}};
+	// Alternative layers
+	if (!validate_layers(default_debug_layers, instance_layers))
+	{
+		default_debug_layers = {
```

Comment:
Can we add a warning here to let the developer know a fallback is being attempted:

`LOGW("Attempted to enable VK_LAYER_KHRONOS_validation but it wasn't found, falling back to VK_LAYER_LUNARG_standard_validation")`

We can do the same for the further fallbacks.

Reply:
I'll try to add that today.